File: app/src/main/java/com/developer/valyutaapp/model/ValuteUpdate.java
package com.developer.valyutaapp.model;


import androidx.room.Entity;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

import io.reactivex.annotations.NonNull;

@Entity
@Root(name="Valute")
public class ValuteUpdate {

    @NonNull
    @Attribute(name = "ID")
    private int id;

    @Element(name = "CharCode")
    public String charCode;

    @Element(name = "Nominal")
    public int nominal;

    @Element(name = "Name")
    public String name;

    @Element(name = "Value")
    public String value;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getCharCode() {
        return charCode;
    }

    public void setCharCode(String charCode) {
        this.charCode = charCode;
    }

    public int getNominal() {
        return nominal;
    }

    public void setNominal(int nominal) {
        this.nominal = nominal;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}

File: app/src/main/java/com/developer/valyutaapp/domain/entities/ValuteUpdate.kt
package com.developer.valyutaapp.domain.entities

import androidx.room.Entity
import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Entity
@Root(name = "Valute")
class ValuteUpdate {
    @Attribute(name = "ID")
    var id = 0

    @Element(name = "CharCode")
    var charCode: String? = null

    @Element(name = "Nominal")
    var nominal = 0

    @Element(name = "Name")
    var name: String? = null

    @Element(name = "Value")
    var value: String? = null
}